apps/labs/posts/intel-oneapi-msvc-spin-scipy.md

```diff
@@ -0,0 +1,174 @@
+This was all handled in the PR, `BUG/CI: Compile and run tests with ifx + MKL on Linux` [[gh-21173]](https://github.com/scipy/scipy/pull/21173), where I also added a CI job to test the combination of `gcc` + `g++` + `ifx` + MKL on Linux, marking another important step toward full Intel oneAPI support for SciPy.
+
+The final milestone for Intel oneAPI support was the CI job for testing SciPy with `icx`, `icpx`, `ifx`, and MKL [[gh-21254]](https://github.com/scipy/scipy/pull/21254).
+In this step, we replaced `gcc` with `icx` and `g++` with `icpx`, and the build successfully passed on Linux. However, on Windows, we were unable to proceed further due to an unresolved import error with `arpack`. Despite several attempts to fix the issue, all solutions led to dead ends. As a result, we decided to pause the effort on Windows for now, as the work required did not seem to justify the outcome.
+
+Through these efforts, we’ve made significant strides in integrating Intel oneAPI with SciPy, ensuring better support for both Linux and Windows platforms.
+While there are still challenges ahead, the progress made provides a solid foundation for future development.
+
+#### Moving away from `dev.py` and migrating to `spin`
+
+Before diving into the details of our work, it’s important to provide some context on **Meson** — the build system we’re using. Meson builds a project in three main steps:
+
+1. **Configuration**: The first step involves running `meson setup`, which configures the project for building. If you’re familiar with CMake, this is similar to calling `cmake .` in a project.
+
+2. **Compilation**: Next, we run `meson compile`, which executes the compiler commands and generates the necessary libraries. For users familiar with `make`, this step is equivalent to running `make` or `make -j8`.
+
+3. **Installation**: Finally, `meson install` installs the compiled libraries and source files (for Python) into the installation directory.
+
+With this context in place, let's dive into the specifics of the work we did.
+
+We decided to transition from `dev.py` to `spin` due to a series of issues reported with `dev.py`. For example:
+
+- [doit interfering with the command history of pdb](https://github.com/scipy/scipy/issues/16452)
+- Emoji rendering issues in the Windows CI environment [[gh-18046](https://github.com/scipy/scipy/issues/18046)]
+- Inability of `dev.py` to resolve Homebrew-installed Python [[gh-18998](https://github.com/scipy/scipy/issues/18998)]
+
+Many of these problems stem from `doit`, which is a dependency of `dev.py`.
+One of the advantages of `spin` is that it has fewer dependencies, which in turn means fewer points of failure.
+While this trade-off sacrifices some of the output’s polish, it greatly improves the reliability of the development process for SciPy.
+
+**Why not just fix `dev.py`?**
+
+You might wonder, "Why not just update `dev.py` to fix these issues? Why switch to a new third-party tool like `spin`?"
```

Comment:
```suggestion
You might wonder, "Why not just update `dev.py` to fix these issues? Why switch to a new third-party tool like `spin`?"
```

Removing since it feels redundant.